.github/workflows/send-alert-emails.yml

```diff
@@ -8,7 +8,8 @@ env:
   SNAPSHOT_ID: ${{ github.event.client_payload.snapshot_id }}
   # Should be 'dev', 'staging', or 'prod'. Determines which Firebase project to use.
   FIREBASE_ENV: ${{ github.event.client_payload.firebase_env }}
-
+  # 'true' or 'false' to indicuate whether we should send the emails or just run (true will send the emails)
+  SEND_EMAILS: ${{ github.event.client_payload.send_emails }}
```

Comment:
There's a tools/send-alert-emails.sh script that you can add this to so we can actually kick off the job.

Reply:
okay cool yes done